FGEGraphics/ClientSystem/ShaderLocations.cs

```diff
@@ -6,6 +6,7 @@
 // hold any right or permission to use this software until such time as the official license is identified.
 //
 
+using FGEGraphics.ClientSystem.ViewRenderSystem;
```

Comment:
accidental import?

Reply:
Yes my bad I must have used a see class thing in the summary at one point but removed it